source/gcp/configuration.rst

```diff
@@ -0,0 +1,125 @@
+| **Default value**  | yes     |
++--------------------+---------+
+| **Allowed values** | yes, no |
++--------------------+---------+
+
+project_id
+^^^^^^^^^^
+
+Put your GCloud project ID here ``(Required)``.
+
++--------------------+-----------------------------+
+| **Allowed values** | String with your project ID |
++--------------------+-----------------------------+
+
+subscription_name
+^^^^^^^^^^^^^^^^^
+
+Put your GCloud subscription name here ``(Required)``.
+
++--------------------+------------------------------------+
+| **Allowed values** | String with your subscription name |
++--------------------+------------------------------------+
+
+credentials_file
+^^^^^^^^^^^^^^^^
+
+Path to credentials file ``(Required)``. Keep in mind that the root would be your ``WAZUH_PATH``. 
+
+.. note:: This file must be in JSON format.
+
++--------------------+------------------------------------------------+
+| **Allowed values** | Relative path to your credentials file         |
++--------------------+------------------------------------------------+
+| **Example**        | credentials.json (WAZUH_PATH/credentials.json) |
++--------------------+------------------------------------------------+
+
+logging
+^^^^^^^
+
+Logger level. The module only shows the messages which have a level as ``log_level`` at least ``(Optional)``. 
+
++--------------------+--------------------------+
+| **Default value**  | 3 (INFO)                 |
++--------------------+--------------------------+
+| **Allowed values** | A number between 1 and 6 |
++--------------------+--------------------------+
+
+There are six levels of debug in this module:
+
++-----------+-------------+
+| **Level** | **Meaning** |
++-----------+-------------+
+| 1         | notset      |
++-----------+-------------+
+| 2         | debug       |
++-----------+-------------+
+| 3         | info        |
++-----------+-------------+
+| 4         | warning     |
++-----------+-------------+
+| 5         | error       |
++-----------+-------------+
+| 6         | critical    |
++-----------+-------------+
+
+max_messages
+^^^^^^^^^^^^
+
+Maximum number of retrieved messages for each request. This means that if there are more messages than this value, there will be another request until the response will be empty (``Optional``).
+
++--------------------+---------------------------+
+| **Default value**  | 100                       |
++--------------------+---------------------------+
+| **Allowed values** | Any number from 1 to 1000 |
++--------------------+---------------------------+
+
+interval
+^^^^^^^^
+
+Interval of time between each pull from GCloud Pub/Sub ``(Required)``.
+
++--------------------+-------------------------------------+
+| **Allowed values** | Positive numbers with the time unit |
++--------------------+-------------------------------------+
+| **Example**        | 10s, 10m, 10h                       |
++--------------------+-------------------------------------+
```

Comment:
Hello team,

One question for you. There is a file that describes the _gcp-pubsub_ block, a description very similar to this file, in this [Pull Request](https://github.com/wazuh/wazuh-documentation/pull/1947/files) I think is better to have only one, don't you? In case you prefer this file, it is very important to include the scheduling options: (_day_, _wday_ and _time_).

Reply:
Hello Daniel,

I agree with you. Since the description from your PR is way better, I will reference it from my `.rst`.

Thanks!